dhcpv6/ztpv6/parse_remote_id.go

```diff
@@ -8,12 +8,32 @@ import (
 	"github.com/insomniacslk/dhcp/dhcpv6"
 )
 
-var (
+var circuitRegexs = []*regexp.Regexp{
```

Comment:
Do you actually need all of the regexes? Are you expecting all of them to have v6 support and be present in a relay? If not, can we only add the ones we are sure about?

Why do we end up having more regexes than v4? Do any of the 2 regexes at the top overlap with some of the newly added ones?

Reply:
Ah, I forgot to consolidate the first one with the 4th.

Looking again at this, I don't think I need any of the changes here, so I'm just going to remove it.